deployment/clowdapp.yaml

```diff
@@ -1497,9 +1488,7 @@ parameters:
     scale_down_cluster,\
     scale_up_inspection_cluster,\
```

Comment:
looks like some of those other tasks that were removed also need to be removed from the worker here.

Reply:
Good catch! I've fixed this in a subsequent push, and I've also added a script to our CI to make sure we never miss something like this again (hopefully).